src/handlers.rs

```diff
@@ -413,6 +421,28 @@ fn handle_read_file_sync(
   ))
 }
 
+fn handle_write_file_sync(
+  d: *const DenoC,
+  builder: &mut FlatBufferBuilder,
+  filename: &str,
+  data: &[u8],
+  perm: u32,
+) -> HandlerResult {
+  debug!("handle_write_file_sync {}", filename);
+  let deno = from_c(d);
+  if deno.flags.allow_write {
+    // TODO(ry) Use perm.
```

Comment:
Yeah that's kinda bad. Maybe `assert(perm & 0o666 === 0o666)` then.

Reply:
https://doc.rust-lang.org/nightly/std/fs/struct.Permissions.html#method.readonly

Reply:
Thanks - could you submit a PR?

Reply:
#567